src/runtime/runtime-constants.ts

```diff
@@ -1,13 +1,20 @@
+/**
+ * Bit flags for recording various properties of VDom nodes
+ */
 export const enum VNODE_FLAGS {
   isSlotReference = 1 << 0,
 
-  // slot element has fallback content
-  // still create an element that "mocks" the slot element
+  /**
+   * slot element has fallback content
+   * still create an element that "mocks" the slot element
+   */
   isSlotFallback = 1 << 1,
 
-  // slot element does not have fallback content
-  // create an html comment we'll use to always reference
-  // where actual slot content should sit next to
+  /**
+   * slot element does not have fallback content
+   * create an html comment we'll use to always reference
+   * where actual slot content should sit next to
+   */
```

Comment:
I know you didn't write this, but is this comment accurate to what this really does? Seems like this is more closely tied to the host node (`s-hn`). Not sure where slot fallback content fits into this and why that's mentioned in the comment.

Reply:
yeah I just changed the formatting to make them visible, but I agree it doesn't make sense...I'll come up with something else